features/web-audio.yml

```diff
@@ -0,0 +1,216 @@
+  - api.AnalyserNode.smoothingTimeConstant
+  - api.AudioBuffer
+  - api.AudioBuffer.AudioBuffer
+  - api.AudioBuffer.copyFromChannel
+  - api.AudioBuffer.copyToChannel
+  - api.AudioBuffer.duration
+  - api.AudioBuffer.getChannelData
+  - api.AudioBuffer.length
+  - api.AudioBuffer.numberOfChannels
+  - api.AudioBuffer.sampleRate
+  - api.AudioBufferSourceNode
+  - api.AudioBufferSourceNode.AudioBufferSourceNode
+  - api.AudioBufferSourceNode.buffer
+  - api.AudioBufferSourceNode.detune
+  - api.AudioBufferSourceNode.loop
+  - api.AudioBufferSourceNode.loopEnd
+  - api.AudioBufferSourceNode.loopStart
+  - api.AudioBufferSourceNode.playbackRate
+  - api.AudioBufferSourceNode.start
+  - api.AudioContext
+  - api.AudioContext.AudioContext
+  - api.AudioContext.baseLatency
+  - api.AudioContext.close
+  - api.AudioContext.createMediaElementSource
+  - api.AudioContext.createMediaStreamDestination
+  - api.AudioContext.createMediaStreamSource
+  - api.AudioContext.createMediaStreamTrackSource
+  - api.AudioContext.getOutputTimestamp
+  - api.AudioContext.outputLatency
+  - api.AudioContext.resume
+  - api.AudioContext.suspend
+  - api.AudioDestinationNode
+  - api.AudioDestinationNode.maxChannelCount
+  - api.AudioListener
+  - api.AudioListener.forwardX
+  - api.AudioListener.forwardY
+  - api.AudioListener.forwardZ
+  - api.AudioListener.positionX
+  - api.AudioListener.positionY
+  - api.AudioListener.positionZ
+  - api.AudioListener.upX
+  - api.AudioListener.upY
+  - api.AudioListener.upZ
+  - api.AudioNode
+  - api.AudioNode.channelCount
+  - api.AudioNode.channelCountMode
+  - api.AudioNode.channelInterpretation
+  - api.AudioNode.connect
+  - api.AudioNode.context
+  - api.AudioNode.disconnect
+  - api.AudioNode.numberOfInputs
+  - api.AudioNode.numberOfOutputs
+  - api.AudioParam
+  - api.AudioParam.automationRate
+  - api.AudioParam.cancelAndHoldAtTime
+  - api.AudioParam.cancelScheduledValues
+  - api.AudioParam.defaultValue
+  - api.AudioParam.exponentialRampToValueAtTime
+  - api.AudioParam.linearRampToValueAtTime
+  - api.AudioParam.maxValue
+  - api.AudioParam.minValue
+  - api.AudioParam.setTargetAtTime
+  - api.AudioParam.setValueAtTime
+  - api.AudioParam.setValueCurveAtTime
+  - api.AudioParam.value
+  - api.AudioParamMap
+  - api.AudioScheduledSourceNode
+  - api.AudioScheduledSourceNode.ended_event
+  - api.AudioScheduledSourceNode.start
+  - api.AudioScheduledSourceNode.stop
+  - api.AudioWorklet
+  - api.AudioWorkletGlobalScope
+  - api.AudioWorkletGlobalScope.currentFrame
+  - api.AudioWorkletGlobalScope.currentTime
+  - api.AudioWorkletGlobalScope.registerProcessor
+  - api.AudioWorkletGlobalScope.sampleRate
+  - api.AudioWorkletNode
+  - api.AudioWorkletNode.AudioWorkletNode
+  - api.AudioWorkletNode.parameters
+  - api.AudioWorkletNode.port
+  - api.AudioWorkletNode.processorerror_event
+  - api.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.port
```

Comment:
I would tend to get rid of this and perhaps add it as a separate feature. To me, being able to run webaudio graphs in a worklet is an nice-to-have optimization technique that doesn't prevent using the API at all, and should be seen as a later addition.

Reply:
I've split out worklets.